### PR TITLE
Test realtime map issue118

### PR DESCRIPTION
**Pull request by amabelleS**
# Description
Added test for realtime map page.

Issues: 
- You won't always see it in the browser/test env, but sometimes the minutes difference in the initial render looks like that: 
image

- Sometimes I needed to do the test repeatedly (from the ui) and even restart my computer (stackoverflow?)
- Mui interface with the test: the date input - could not navigate to last month, the hour input - need to add test to that? I tried, I put my tries on a comment in the code...

### CODE REVIEW

File: src/pages/RealtimeMapPage.tsx [hunk 1]
```diff
@@ -47,7 +47,7 @@ export function numberToColorHsl(i: number, max: number) {
 }
 
 const fiveMinutesAgo = moment().subtract(5, 'minutes')
-const fourMinutesAgo = fiveMinutesAgo.add(1, 'minutes')
+const fourMinutesAgo = moment().subtract(4, 'minutes')
```
Comment (NoamGaash) [inline src/pages/RealtimeMapPage.tsx:50]: did you try revert this change? @amabelleS

File: package.json [hunk 2]
```diff
@@ -3,6 +3,7 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@babel/runtime": "^7.23.2",
```
Comment (shootermv) [inline package.json:6]: may be im wrong here,
but
as far as know -
babel is something that replaced by vite
and thus  - no more needed

File: src/pages/RealtimeMapPage.tsx (hunk 1 above)
Comment (amabelleS) [reply src/pages/RealtimeMapPage.tsx:50]: > did you try revert this change? @amabelleS

I'm not sure what you mean.. my code is, from the last PR: 
```const fiveMinutesAgo = moment().subtract(5, 'minutes')
const fourMinutesAgo = moment().subtract(4, 'minutes')```

this line: ```const fourMinutesAgo = fiveMinutesAgo.add(1, 'minutes')``` is from the version before my last PR. 
There was a bug, in the initial render fourMinutesAgo & fiveMinutesAgo both got the value of 4 minutes ago. 
I did not revert it:( If that means that I chose to reintroduce this bug.

I'm a GitHub merge conflicts newbie:) relatively...

File: package.json (hunk 2 above)
Comment (amabelleS) [reply package.json:6]: > maybe I'm wrong here, but as far as know - babel is something that replaced by vite and thus - no more needed

There was a conflict with the versions: 7.23.1 (my branch) // 7.23.2 (main). 
I don't think you are wrong. 
Could the yarn.lock file be different in different environments of the same project? How come my version was not updated when I run ```yarn install``` before starting to code? 

I had some bugs that I tried to resolve alone, but tell me if it caused bugs that I'm not aware of, please 🙏

**Comment by amabelleS** [comment]
> Please see the conflicts, Thanks!!! 🙏

I only see the conflict in the yarn.lock file. Tried to resolve it by installing the latest version of @babel/runtime. But it just added the last version, with all the old ones remaining.. from what I understand here. 

Any suggestions on how to resolve this, or should I try to google it?:)

File: package.json (hunk 2 above)
Comment (NoamGaash) [reply package.json:6]: @amabelleS 
https://stackoverflow.com/questions/2364147/how-to-get-just-one-file-from-another-branch
please take `package.json` and `yarn-lock.json` from `main`

**Comment by NoamGaash** [comment]
`git merge --strategy-option theirs` will solve the conflict
https://stackoverflow.com/a/10697551/3237334

File: src/pages/RealtimeMapPage.tsx (hunk 1 above)
Comment (NoamGaash) [reply src/pages/RealtimeMapPage.tsx:50]: I believe the `const fourMinutesAgo = fiveMinutesAgo.add(1, 'minutes')` is better, and asking you to revert the change you're making here.
When you construct a new `Moment`, it can make a little gap in time, so the difference won't be exactly one minute.
image

Comment (amabelleS) [reply src/pages/RealtimeMapPage.tsx:50]:
```
const fiveMinutesAgo = moment().subtract(5, 'minutes')
const fourMinutesAgo = moment(fiveMinutesAgo).add(1, 'minutes')
```
This fixes this bug.

File: package.json (hunk 2 above)
Comment (amabelleS) [reply package.json:6]: Please check this PR: 
https://github.com/hasadna/open-bus-map-search/pull/164

I opened a new branch on a new clone, fresh start with hopefully fewer bugs:)

Comment (amabelleS) [reply package.json:6]: I am closing this PR because we have the same changes in #164
